okhttp/src/main/java/okhttp3/ResponseBody.kt

```diff
@@ -121,16 +121,31 @@ abstract class ResponseBody : Closeable {
    * possibility for your response.
    */
   @Throws(IOException::class)
-  fun bytes(): ByteArray {
+  fun bytes() = consumeSource(BufferedSource::readByteArray) { it.size }
```

Comment:
Can't be function reference because of https://youtrack.jetbrains.com/issue/KT-21014